prow/plugins/require-matching-label/require-matching-label.go

```diff
@@ -155,6 +169,32 @@ func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) error {
 	}
 	return handle(pc.Logger, pc.GitHubClient, cp, pc.PluginConfig.RequireMatchingLabel, e)
 }
+func handleCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent) error {
+	return handleComment(pc, &ce)
+}
+func handleComment(pc plugins.Agent, ce *github.GenericCommentEvent) error {
+	//consider open and close PRs and new comments
+	if ce.IssueState != "open" || ce.IssueState != "closed" || ce.Action != github.GenericCommentActionCreated {
```

Comment:
Should it be `(ce.IssueState != "open" && ce.IssueState != "closed") || ce.Action != github.GenericCommentActionCreated`?

Reply:
Can we also add some unit tests under `prow/plugins/require-matching-label/require-matching-label_test.go`?